### ISSUE TITLE
Change loop playback functionality?

### ISSUE THREAD

**Issue by nielmin** [OxygenCobalt/Auxio#13]
#### Describe the feature you want to implement:

I'm on version 1.3.3, and find the loop playback implementation unorthodox compared to other audio players.
When toggled:
 - \[  \] loops a single song indefinitely.
 - \[ 1 \] loops the song once then continues playing the next song.

Not to mention, the option to loop the whole playlist is hidden in the settings.

#### Is your feature request related to a problem? Please describe:
No.

#### Why do you think this will improve everyone's usage of Auxio?

I think it would be better to have [ ] loop the entire playlist and [1] to loop a single song indefinitely when toggled. It doesn't really make sense to have a dedicated toggle ([1]) loop a song one time only. This would also tidy up the settings page.
#### Due Diligence:

- [x] I have read  [Accepted Additions and Requests](https://github.com/OxygenCobalt/Auxio/blob/dev/info/ADDITIONS.md) and the [Contribution Guidelines](https://github.com/OxygenCobalt/Auxio/blob/dev/.github/CONTRIBUTING.md).

**Comment by OxygenCobalt** [comment]
Oh.

I modeled the loop system off of [Music Player GO](https://github.com/enricocid/Music-Player-GO) which was the player I used to use before building Auxio. Since in retrospect I don't really benefit from looping a song once either, I'll implement this change.

As for the behavior at the end of a playlist, I'll likely default to the Spotify implementation where:

Loop Off -> Loop playlist, but also pause
Loop On -> Loop playlist and don't pause

Thanks for letting me know about this, I really want to make sure this player does not have annoying UI quirks like this so I appreciate you pointing this out.
